bpf/lib/nat.h

```diff
@@ -465,9 +465,7 @@ static __always_inline __maybe_unused int snat_v4_create_dsr(struct __ctx_buff *
 		tuple.sport = l4hdr.dport;
 		break;
 	default:
-		// NodePort svc can be reached only via TCP or UDP, so
-		// drop the rest
-		return DROP_NAT_UNSUPP_PROTO;
+		return NAT_PUNT_TO_STACK;
```

Comment:
`NAT_PUNT_TO_STACK` is `< 0`, so the caller of `snat_v4_create_dsr()` will drop such packet: https://github.com/cilium/cilium/blob/master/bpf/lib/nodeport.h#L819-L820

Reply:
Hm, but that is okay no? We know from handle_dsr_v4() that there was an IP option for DSR present, so anything else would be bad not to drop. I'll leave it at `DROP_NAT_UNSUPP_PROTO` here.